### DISCUSSION TITLE
Can we have a fix for base64 regular expression ?

### DISCUSSION THREAD

**Discussion by yogesharora388** [singerdmx/flutter-quill#1691] [Desktop]
It should be
r'^(?:[A-Za-z0-9+\/][A-Za-z0-9+\/][A-Za-z0-9+\/][A-Za-z0-9+\/])*(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=|[A-Za-z0-9+\/]{4})$'

instead of r'^(?:[A-Za-z0-9+\/]{4})*(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=|[A-Za-z0-9+\/]{4})$'

FilePath: /flutter_quill_extensions-9.2.6/lib/utils/patterns.dart

**Comment by yogesharora388** [comment]
I am not able to show base64 image in release mode. Screen is greying out

**Comment by EchoEllet** [comment]
Hi, can you explain why this is a fix? also it might be helpful to send a PR since it was your idea even though I can do it on my own but I just need more info

**Reply by yogesharora388** [reply]
<img width="799" alt="Screenshot 2024-01-23 at 1 04 00 PM" src="https://github.com/singerdmx/flutter-quill/assets/133763051/991d77e5-6ec7-401f-a848-7cd051216a53">

Here below are steps-

1. I added an image in an editor.
2. Saved the file and replaced the image path with base64 string of image content. (It means I am not saving the image path in image tag under insert tag. Instead of image path I am reading the content and converted into base64 string and saving it.)
3. When I am trying to open the same file in my editor then it is greying out. But it is only happening in release build not in debug more.


I hope it will helpful to understand the issue.

**Reply by yogesharora388** [reply]
I am not able to push my PR because of permission error.

**Reply by yogesharora388** [reply]
@ellet0 Can we please check this ?

**Reply by EchoEllet** [reply]
> @ellet0 Can we please check this ?

Sure thing
